docs/source/coredev/release_process.rst

```diff
@@ -170,6 +170,12 @@ Get a fresh clone of the tag for building the release::
 8. Run the release script
 -------------------------
 
+.. important::
+
+    Following releases instructions have information to release IPython 5.x and
```

Comment:
These steps cover instructions for creating releases of IPython 5.x LTS and IPython 6.x. Ignore release steps for Python 2 when releasing IPython 6.x which no longer supports Python 2.